docs/input-kafka.asciidoc

```diff
@@ -183,6 +183,8 @@ for the initial connection to discover the full cluster membership (which may ch
 so this list need not contain the full set of servers (you may want more than one, though, in
 case a server is down).
 
+NOTE: Kafka client does not support proxy setup. Only schema registry supports proxy.
```

Comment:
@kaisecheng I think we need improved wording, e.g.:

This plugin only supports using a proxy when communicating to the Schema Registry <this should link to the setting>. This plugin does not support using a proxy when communicating to Kafka. This is because of a limitation in the Kafka Client, which this plugin uses. For more information, please refer to the upstream documentation <this should link to the documentation>.

@kaisecheng also, I'm, guessing some of this wording / note is required for the Kafka output.

@karenzone I think this should be in the special format to render notes, right?

Reply:
You're right! We do need to use special formatting to render notes correctly.  Kaise nailed it in the original commit.  :-)